onnxruntime/python/tools/transformers/models/llama/README.md

```diff
@@ -0,0 +1,161 @@
+# LLaMA-2
+
+## Exporting LLaMA-2
+
+There are several ways to export LLaMA-2 models (using LLaMA-2 7B as an example).
+
+Option 1: from convert_to_onnx
+```
+# From source:
+$ git clone https://github.com/microsoft/onnxruntime
+$ cd onnxruntime/onnxruntime/python/tools/transformers/models/llama
+$ python3 convert_to_onnx.py -m meta-llama/Llama-2-7b-hf --output llama2-7b
+
+# From wheel:
+$ python3 -m onnxruntime.transformers.models.llama.convert_to_onnx -m meta-llama/Llama-2-7b-hf --output llama2-7b
+```
+
+To make this option compatible with [Hugging Face's Optimum](https://github.com/huggingface/optimum), you will need to create `config.json` and `generation_config.json` for your model and store them in the same directory as your ONNX models. For example, you can find those JSON files for LLaMA-2 7B on Hugging Face [here](https://huggingface.co/meta-llama/Llama-2-7b-hf).
+
+Option 2: from [Microsoft's custom export](https://github.com/microsoft/Llama-2-Onnx)
+
+Option 3: from [Hugging Face's Optimum](https://github.com/huggingface/optimum)
+
+## Examples of Exporting LLaMA-2
+
+Here are some additional examples for exporting LLaMA.
+
+### Export Saved Model on Disk
+```
+# From source:
+$ python3 convert_to_onnx.py -m meta-llama/Llama-2-7b-hf --input ./Llama-2-7b-hf --output ./llama2-7b
+
+# From wheel:
+$ python3 -m onnxruntime.transformers.models.llama.convert_to_onnx -m meta-llama/Llama-2-7b-hf --input ./Llama-2-7b-hf --output ./llama2-7b
+```
+
+### Export with Different Precision
+
+FP16:
+```
+# From source:
+$ python3 convert_to_onnx.py -m meta-llama/Llama-2-7b-hf --output llama2-7b --precision fp16
+
+# From wheel:
+$ python3 -m onnxruntime.transformers.models.llama.convert_to_onnx -m meta-llama/Llama-2-7b-hf --output llama2-7b --precision fp16
+```
+
+INT8:
+```
+# From source:
+$ python3 convert_to_onnx.py -m meta-llama/Llama-2-7b-hf --output llama2-7b --precision int8 --smooth_quant
```

Comment:
when the output folder is same for fp32 and int8. This will remove previously exported fp32 models. I suggest to use different folder name for each data type like `--output llama2-7b-int8`

Reply:
Both the FP32 and INT8 models are created and saved in the output folder when running this command. The script removes the models so that the data file doesn't double in size if the same export command is run multiple times. I'll change the folder names for each data type.